HttpClientGuidance.md

```diff
@@ -16,7 +16,7 @@
 
 ## A note about WebClient
 
-WebClient is considered a legacy .NET API at this point and has been completely superseded by HttpClient. No new code should be written with HttpClient.
+WebClient is considered a legacy .NET API at this point and has been completely superseded by HttpClient. No new code should be written with WebClient.
```

Comment:
There is a 'No' at the start of this sentence; I believe it already says what you think you're making it say by changing it ;)

Reply:
"New code should be written with HttpClient"  != "`No` new code should be written with HttpClient"
So maybe "No new code should be written with WebClient" is better😄.